usaspending_api/api_contracts/contracts/v2/search/spending_by_award.md

```diff
@@ -172,6 +172,8 @@ List of table columns
 + `set_aside_type_codes`: `NONE` (optional, array[string])
 + `extent_competed_type_codes`: `A` (optional, array[string])
 + `tas_codes` (optional, array[TASCodeObject], fixed-type)
++ `object_class` (optional, array[string])
```

Comment:
Looking through `view_selector.py` and the usage of `get_view_queryset()` there are a lot more API Contracts that could be updated to reflect this new filter.

Reply:
Do we want to be announcing uses of endpoints we haven't worked with or tested?

Reply:
To me, it boils down to the purpose of the API Contracts. I think of it as a way to capture the current functionality of the endpoint regardless of whether or not it is used.

Reply:
I think of it as a document of promises we intend to keep. If we had been documenting the v1 endpoints, would you have included all the possible joins you could use, or just the ones used by the app we were comfortable supporting?

Reply:
(this might quickly become a discussion to move off of github)